posts/superseding-bpmodels/index.qmd

```diff
@@ -22,25 +22,27 @@ knitr::opts_chunk$set(
 
 For {bpmodels}, we wanted to unify the simulation functions (existing as two functions previously) and improve the function signature by renaming several of the arguments for readability. We also wanted to introduce an object-oriented workflow to aid in interoperability with existing tools such as [epicontacts](https://www.repidemicsconsortium.org/epicontacts/) and [epiparameter](https://epiverse-trace.github.io/epiparameter/). The object-oriented backend would also allow us to implement better methods for printing, summarising and aggregating the simulation output. Some of these considerations would have been less disruptive than others but the change in function name and signature would have led to a lot of disruptions including deprecating the existing functions and arguments.
 
 One thing is clear from the examples on scope changes - they often lead to name changes. Another important decision was whether to rebrand the package with a new name. A new name can signal a fresh approach and address limitations of the original package. The most popular example is the renaming of `{ggplot}` to [`{ggplot2}`](https://ggplot2.tidyverse.org/index.html). Other examples include renaming {reshape} to {reshape2} and {tidyr}. In our case, we decided to fork the original [`{bpmodels}`](https://epiforecasts.github.io/bpmodels/) repository under Epiverse-TRACE and maintain the old package to avoid breaking scripts that rely on it. At the same time, we introduced {epichains} as the successor. The name reflects the fact that it is a package for analysing *epidemiological transmission chains*.
 
-We also had to consider whether to archive {bpmodels} or allow it to coexist with {epichains}. We decided to keep {bpmodels} accessible to sustain the reproducibility of existing code using the package. The package was moved back to the [epiforecasts GitHub organisation](https://epiforecasts.github.io/bpmodels/) where it originated from. We, however, added a lifecycle badge to communicate the package’s retired status and text in the README about our plans for low maintenance.
+A second key consideration is the plans that the original package author(s) may have had and their views on any future changes. In our case this was fairly straightforward because the maintainer of the original package was fully involved in the refactoring. Another package author who had made substantial contributions could be reached and approval sought. More generally, however, bringing all package authors on board with things like scope change, name change etc. is an important step in taking on maintenance of a package and one that must not be neglected.
```

Comment:
```suggestion
A second key consideration are the plans that the original package author(s) may have had and their views on any future changes. In our case this was fairly straightforward because the maintainer of the original package was fully involved in the refactoring. Another package author who had made substantial contributions could be reached and gave approval. More generally, however, bringing all package authors on board with, e.g., changes in scope and name is an important step in taking on maintenance of a package and one that should not be neglected.
```